pkg/resources/common.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+const FullyQualifiedNameAttributeName = "fully_qualified_name"
```

Comment:
we have now multiple places with common attribute names, maybe we should have common_attribute_names.go or something like this?